codice/ddf-ui
# Convert gazetteer autocomplete and add downstream suggestion extension

Converts the gazetteer autocomplete to use a mui autocomplete and fixes the debounce.

Adds an extension point for downstream suggestions

Testing:

1. Navigate to search and type in the gazetteer.
1. Verify that you get relevant results.
1. Make sure things work as expected. 

![image](https://user-images.githubusercontent.com/22667297/101699526-85111180-3a38-11eb-9fdb-9cd72dc228bc.png)
